### PR TITLE
vendor updates for servicecatalog

### PR DESCRIPTION
**Pull request by bw-intuit**
First of series of PRs to add AWS service catalog support.

More info: https://github.com/terraform-providers/terraform-provider-aws/pull/1663

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -661,6 +661,12 @@
 			"versionExact": "v1.10.44"
 		},
 		{
+			"checksumSHA1": "TsLkr/jtNxQY3RSJjbCZXEaopG0=",
+			"path": "github.com/aws/aws-sdk-go/service/servicecatalog",
+			"revision": "f43761405cff341da21289c4f14800c41ccdc10b",
+			"revisionTime": "2017-08-25T16:44:15Z"
```
Comment (Ninir) [inline vendor/vendor.json]: I cannot see the version here, like `v1.10.44`
Can you please provide the govendor command you used?

Thanks! 😄

**Comment by bw-intuit** [comment]
@ninir I just used:

```
govendor add github.com/aws/aws-sdk-go/service/servicecatalog
```

Do you have a recommended cmd?

Thanks

**Comment by Ninir** [comment]
Would be better to use `govendor fetch github.com/aws/aws-sdk-go/service/servicecatalog@=v1.10.44`, so that it fetches the exact version and lock it.

I often use this [cheatsheet](https://github.com/kardianos/govendor/wiki/Govendor-CheatSheet) to better handle vendoring.

Thanks @bw-intuit  :)

**Comment by bw-intuit** [comment]
@Ninir gotcha, updated with:

```
govendor fetch github.com/aws/aws-sdk-go/service/servicecatalog@v1.10.46
```

**Comment by Ninir** [comment]
@bw-intuit As you specified 1.10.46, we need to first update other vendors to 1.10.46, so that every vendor has the same version. Let me open the PR for that, and I'll merge this one afterwards :)

(or you can use 1.10.44 directly, so that it's easier :) )

**Comment by bw-intuit** [comment]
1.10.44 should be fine, will change.  Thanks

**Comment by bw-intuit** [comment]
@Ninir thanks, Updated first resource here: https://github.com/terraform-providers/terraform-provider-aws/pull/1694

**Pull request merged**